src/cloud/project/ece-tools-update.md

```diff
@@ -8,7 +8,24 @@ functional_areas:
   - Cloud
   - Upgrade
 ---
-Typically, an _update_ includes patches and available hotfixes as part of the `magento-cloud-metapackage`. If you still use a version of {{site.data.var.ece}} that does not contain the `{{site.data.var.ct}}` package, then your project requires an [upgrade]({{ site.baseurl }}/cloud/project/ece-tools-upgrade-project.html).
+
+When you update {{site.data.var.ct}}, you also update to the latest version of the following {{site.data.var.csuite}} packages, which are dependencies for the `{{site.data.var.ct}}` package:
+
+-  **`{{site.data.var.mcp}}`**–A set of patches which improve the integration of all Magento versions with Cloud environments. This package includes Magento patches and available hot fixes that are applied when you use `{{site.data.var.ct}}` to deploy to Staging and Production environments.
+
+-  **`{{site.data.var.mcd}}`**–Functionality and Docker images to deploy Magento Commerce to a local Cloud environment.
```

Comment:
this repo does not have Docker images. It just contains configuration for images.